prime-router/src/main/kotlin/fhirengine/azure/FHIRFunctions.kt

```diff
@@ -247,6 +247,7 @@ class FHIRFunctions(
                 ReportStreamEventName.PIPELINE_EXCEPTION,
                 report,
                 fhirEngine.taskAction,
+                null,
```

Comment:
I'm thinking this should not be a required param, but optional param that is set similar to how "params" is set below.